view/script/patient.js

```diff
@@ -1,5 +1,9 @@
 (function patient () {
     'use strict';
+
+    var isNewPatient = window.location.search;
+    var re = /newPatient=true/;
```

Comment:
@poojaRal Its Regex. We can name it whatever we want it to be. What do you suggest ?